features/features.go

```diff
@@ -538,6 +538,14 @@ var (
 						enableForClusterProfile(SelfManaged, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						mustRegister()
 
+	FeatureGateNewOLMPreflightPermissionChecks = newFeatureGate("NewOLMPreflightPermissionChecks").
+							reportProblemsToJiraComponent("olm").
+							contactPerson("tshort").
+							productScope(ocpSpecific).
+							enhancementPR("https://github.com/openshift/enhancements/pull/1768").
+							enableForClusterProfile(SelfManaged, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Comment:
Deliberately not enabling in HyperShift for tech preview? Is the feature not available on HyperShift clusters?

Reply:
Deliberately, yes.  
This is another `NewOLM`-child feature, and the "parent" `NewOLM` feature is not available for Hypershift presently.